packages/core/package.json

```diff
@@ -37,42 +37,40 @@
     "graphql": "15.5.1",
     "graphql-tag": "2.12.5",
     "glob": "7.1.7",
-    "@babel/runtime-corejs3": "7.14.0",
+    "@babel/runtime-corejs3": "7.14.7",
     "x----x----x": "^0 Bins",
-    "esbuild": "0.12.5",
+    "esbuild": "0.12.15",
     "typescript": "4.3.5",
     "cross-env": "7.0.3",
     "x----x----x": "^0 Webpack",
-    "webpack": "4.46.0",
-    "webpack-bundle-analyzer": "3.6.1",
-    "webpack-cli": "3.3.11",
-    "webpack-dev-server": "3.10.3",
-    "webpack-manifest-plugin": "3.0.0",
-    "webpack-merge": "5.1.2",
-    "webpack-retry-chunk-load-plugin": "1.4.0",
+    "webpack": "5.44.0",
+    "webpack-bundle-analyzer": "4.4.2",
+    "webpack-cli": "4.7.2",
+    "webpack-dev-server": "4.0.0-beta.3",
```

Comment:
It's weird to me that they bumped some version beyond 5, but not all of them 🤷‍♂️

Reply:
It's infuriating, actually. For this case re: `webpack-dev-server`, v3 is _technically_ webpack 5 compatible, however there's a known bug so you have to use the v4-beta. This means there are no docs for the breaking changes in dev-server v4, you have to dig through the changelogs yourself. Pain. Ful.

I was talking to Tom about all this today (just giving him an update). I don't really have a problem with Webpack. And I truly think v5 is an improvement. And even more so now I see the value Redwood adds because it internally handles these things. BUT I think in the future we have an opportunity to better manage (and even constrain) our own use of Webpack. E.g. the current changes to `yarn rw dev` is a great de-coupling choice. Also, we currently have too many potential layers of extended config, each including conditionals, that make it challenging to understand what Webpack is doing in a given context (and therefore what plugins/config are being applied). tl;dr: I think we'll be happy about Webpack if we:
- use it less
- simplify (and bound) our use

Thinking out loud here as it's fresh. Not suggesting we do any of this prior to v1.